test/spec/spectest.cpp

```diff
@@ -102,23 +95,62 @@ SpecTest::CommandID resolveCommand(std::string_view Name) {
 
 // Helper function to parse parameters from json to vector of value.
 std::pair<std::vector<WasmEdge::ValVariant>, std::vector<WasmEdge::ValType>>
-parseValueList(const rapidjson::Value &Args) {
+parseValueList(const simdjson::dom::array &Args) {
   std::vector<WasmEdge::ValVariant> Result;
   std::vector<WasmEdge::ValType> ResultTypes;
-  Result.reserve(Args.Size());
-  ResultTypes.reserve(Args.Size());
-  for (const auto &Element : Args.GetArray()) {
-    const auto &Type = Element["type"].Get<std::string>();
-    const auto &ValueNode = Element["value"];
-    if (ValueNode.IsString()) {
-      const auto &Value = ValueNode.Get<std::string>();
+  Result.reserve(Args.size());
+  ResultTypes.reserve(Args.size());
+  for (simdjson::dom::object Element : Args) {
```

Comment:
The `Element` is read-only, use `const simdjson::dom::object &Element` instead.

Reply:
Ok, will make this change as well.